src/Microsoft.AspNet.OData.Shared/Formatter/Deserialization/ResourceSetType.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.AspNet.OData.Formatter.Deserialization
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public enum ResourceSetType
```

Comment:
>public [](start = 4, length = 6)

shall we expose it? or make it internal

Reply:
I think public is fine.

---
In reply to: [512291191](https://github.com/OData/WebApi/pull/2275#discussion_r512291191) [](ancestors = 512291191)